server/tests/helpers/utils.js

```diff
@@ -23,6 +26,31 @@ export const signupUser = agent => new Promise((resolve) => {
     });
 });
 
+export const loginUser = agent => new Promise((resolve) => {
+  agent
+    .post('/api/v1/auth/signup')
+    .send(getUserData({
+      email: 'janet@email.com', username: 'janetgt', password: '123456abcdef'
+    }))
+    .then((res) => {
+      const userData = res.body.data;
+      const token = tokenGenerator(userData.id, false, TOKEN_EXPIRY_DATE, SECRET);
+      agent
+        .get(`/api/v1/auth/verify?token=${token}`)
```

Comment:
I thought this helper was going to be split into 2

Reply:
Yes, it has, as this helper is only doing one thing, so I will just rename it to be verified user, so it doesn't clash with the other user which as being split in Tosin's bug fix PR